### PR TITLE
cmd ref: describe updated `--out` behavior in get/import/*-url

### PR DESCRIPTION
**Pull request by efiop**
Related to iterative/dvc#2610

**IMPORTANT NOTES** (please read, then delete):

- Have you followed the guidelines in our
  [Contributing Documentation](https://dvc.org/doc/user-guide/contributing-docs)?

- The PR title should start with "Fix #bugnum: " (if applicable), followed by a
  clear one-line present-tense summary of the changes introduced in the PR. For
  example: "Fix #bugnum: Introduce the first version of the collection editor.".

- Please make sure to mention "Fix #bugnum" (if applicable) somewhere in the
  description of the PR. This enables GitHub to link the PR to the corresponding
  bug.

### CODE REVIEW

File: static/docs/command-reference/get-url.md [hunk 1]
```diff
@@ -24,8 +24,9 @@ location into the current working directory, regardless of whether it's a DVC
 project. The `dvc get-url` command helps the user do just that.
 
 The `url` argument should provide the location of the data to be downloaded,
-while `out` can be used to specify the (path and) file name desired for the
-downloaded data file or directory.
+while `out` can be used to specify the directory and/or file name desired for
+the downloaded data file or directory. If an existing directory is specified
+(without a file name), then the output will be placed inside of it.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/get-url.md]: We can shorten to "...directory and/or file name desired for the downloaded data. If an existing directory is specified, then..."

File: static/docs/command-reference/get.md [hunk 2]
```diff
@@ -45,9 +45,11 @@ created in the current working directory, with its original file name.
 
 ## Options
 
-- `-o`, `--out` - specify a path (directory and file name) to the desired
+- `-o`, `--out` - specify a path (directory and/or file name) to the desired
   location to place the imported data in. The default value (when this option
   isn't used) is the current working directory (`.`) and original file name.
+  If an existing directory is specified (without a file name), then the output
+  will be placed inside of it.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/get.md]: Similarly, no need for "(without a file name)".

File: static/docs/command-reference/import-url.md [hunk 3]
```diff
@@ -31,8 +31,10 @@ location into the workspace, such that it will be automatically updated (by
 
 The `dvc import-url` command helps the user create such an external data
 dependency. The `url` argument specifies the external location of the data to be
-imported, while `out` can be used to specify the (path and) file name desired
-for the imported data file or directory in the workspace.
+imported, while `out` can be used to specify the directory and/or file name
+desired for the imported data file or directory in the workspace. If an existing
+directory is specified (without a file name), then the output will be placed
+inside of it.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import-url.md]: Let's make this exactly the same as whatever ends up being in static/docs/command-reference/get-url.md? Both should be wrapped to 80 chars.

File: static/docs/command-reference/import.md [hunk 4]
```diff
@@ -63,9 +63,11 @@ downloaded data artifact from the external DVC repository.
 
 ## Options
 
-- `-o`, `--out` - specify a path (directory and file name) to the desired
+- `-o`, `--out` - specify a path (directory and/or file name) to the desired
   location to place the imported data in. The default value (when this option
   isn't used) is the current working directory (`.`) and original file name.
+  If an existing directory is specified (without a file name), then the output
+  will be placed inside of it.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import.md]: Same as in static/docs/command-reference/get.md

File: static/docs/get-started/install.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # Install
 
 There are a few recommended ways to install DVC: OS-specific package/installer,
-`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these may
-be not available.)
+`pip`, `conda` (Anaconda), and Homebrew. (Depending on your OS, some of these
```
Comment (efiop) [inline static/docs/get-started/install.md]: This one was added automatically by the formatter. Let's leave it here, if it is okay with you guys :)

**Pull request merged**